avidemux_de.ts

```diff
@@ -99,130 +102,136 @@ If you re encode video we should drop these informations,
 If you just copy the video without reencoding,
  you should keep them.
 Drop timing informations ?</source>
-        <translation type="unfinished"></translation>
+        <translation>Einige Timing-Informationen sind fehlerhaft.
+Dies kommt bei manchen Aufnahmeprogrammen (Capturing) vor.
+Wenn Sie das Video neukodieren möchen, sollten diese Informationen gelöscht werden,
+ andernfalls wird dies fehlende Bilder/ruckelndes Video verursachen.
+Wenn Sie das Video nur kopieren möchten, anstatt es neuzukodieren,
+ sollten Sie die Informationen beibehalten.
+Timing-Informationen löschen?</translation>
     </message>
 </context>
 <context>
     <name>CalculatorDialog</name>
     <message>
         <location filename="../ADM_userInterfaces/ADM_dialog/calculator.ui" line="+16"/>
         <source>Calculator</source>
-        <translation type="unfinished">Video-Kalkulator</translation>
+        <translation>Video-Kalkulator</translation>
```

Comment:
Same, dead code, no use in touching translation.

Reply:
Will this code never be re-enabled? I have translated it for this case. In some other translation, this context was also translated (maybe when the code was still active)